open-hackathon-server/src/hackathon/config_sample.py

```diff
@@ -17,6 +17,8 @@
 Config = {
     "environment": "local",
     "endpoint": HACKATHON_SERVER_ENDPOINT,
+    "pre_allocate_enabled" : True,
+    "pre_allocate_num" : 100,
```

Comment:
why turn the hackathon-based config to a global config? If we enable this on production, all hackathons will start to pre-callocation including those already ended

Reply:
Because there is nowhere to place those customized items, being here is better than hard code, we can remove it as long as it can be configured somewhere else, like frontend?

Reply:
前端需要跟上，没有的话直接改数据库吧。。。但放到全局似乎有问题,会影响所有hackathon

Reply:
数据库不好改啊，而且改也是hardcode，在config.py中还算是customizable，直接hardcode就更low了，而且也影响所有的hackathon

Reply:
改数据库的某一条记录就行，不会影响其他活动。本来数据库里的值也是通过前端设置进去的，只是界面还没做好。
暂时放到config.py，你至少要加一点改动，只对某些活动生效。或者一放上去，所有的活动包括什么test之类的，全都会pre-allocate....瞬间爆掉啊

Reply:
我大概明白了，后台手动改mongo里的数据记录，我试试看。